f/medical/fn_famAddBandageAction.sqf

```diff
@@ -24,8 +24,8 @@ private _bdgCodeStart = {
 	};
 
 	if (stance _caller == "PRONE") then {
-		// Rifle
-		if (currentWeapon _caller == primaryWeapon _caller && {primaryWeapon _caller != ""}) exitWith {
+		// Rifle or Binocular
+		if ((currentWeapon _caller == binocular _caller) || (currentWeapon _caller == primaryWeapon _caller && {primaryWeapon _caller != ""})) exitWith {
```

Comment:
Would it be worth extracting this to a common variable (e.g. `_callerHoldingRifleBinos`), instead of having this as a repeated check?